pkg/cmd/importcmd/add_collabotor.go

```diff
@@ -63,7 +63,11 @@ func (o *ImportOptions) AddAndAcceptCollaborator(newRepository bool) error {
 	// If the user creating the repo is not the pipeline user, add the pipeline user as a contributor to the repo
 	if pipelineUserName != "" && pipelineUserName != userName { // TODO: not sure why:  && o.ScmFactory.GitServerURL == o.PipelineServer {
 		// Make the invitation
-		_, _, _, err = scmClient.Repositories.AddCollaborator(ctx, fullRepoName, pipelineUserName, permission)
+		alreadyMember := false
+		_, alreadyMember, _, err = scmClient.Repositories.AddCollaborator(ctx, fullRepoName, pipelineUserName, permission)
+		if alreadyMember {
```

Comment:
I am not super familiar with this part of the codebase, but are we sure that gitlab is the only one that does true? How about adding another condition if gitkind is gitlab, then return nil, else check the error ... wdyt?

Reply:
Also probably better to check error first before other things ...